test/node-test/debug.js

```diff
@@ -62,11 +62,15 @@ test('debug#fetch', { skip }, async t => {
   )
   const chunks = []
   const assertions = [
+    /(FETCH [0-9]+:) (fetch has started)/,
     /(FETCH [0-9]+:) (connecting to)/,
+    /(FETCH [0-9]+:) (fetch has received)/,
     /(FETCH [0-9]+:) (connected to)/,
     /(FETCH [0-9]+:) (sending request)/,
     /(FETCH [0-9]+:) (received response)/,
     /(FETCH [0-9]+:) (trailers received)/,
+    /(FETCH [0-9]+:) (fetch has received)/,
+    /(FETCH [0-9]+:) (fetch has received)/,
     /^$/
```

Comment:
why do we receive three times the aend message?

Reply:
It is not the end message returning 3 times, it is the `end` event and `asyncStart` events, We have been discussing if it is necessary to include an `asyncStart` and `asyncEnd` call in 14, which results in the second call you see.